.github/workflows/kubernetes-sync.yml

```diff
@@ -38,22 +32,20 @@ jobs:
         run: |
           cd rancher-k8s
           $GITHUB_WORKSPACE/rancher-k8s/scripts/create-release-branch.sh
-        env:
```

Comment:
Why are we removing `PAT_TOKEN` here but not replacing it with `app-token.outputs.token`?

Reply:
Before, we used to create branches at this step. Now, since we have a larger self-hosted runner, branches are created after the CI passes. This change means the GitHub token is no longer needed for this step.